test/extended/images/extract.go

```diff
@@ -76,7 +76,7 @@ var _ = g.Describe("[Feature:ImageExtract] Image extract", func() {
 			mkdir -p /tmp/test
 			oc image extract --insecure %[2]s/%[1]s/1:busybox --path=/:/tmp/test
 			[ -d /tmp/test/etc ] && [ -d /tmp/test/bin ]
-			[ -f /tmp/test/bin/ls ] && /tmp/test/bin/ls /tmp/test
+			[ -f /tmp/test/bin/ls ]
```

Comment:
hmm... is there something else we can do to confirm the content was extracted successfully?  perhaps extract a text file and confirm the content?  i'd prefer to have some assurance we are not corrupting or truncating the content on extraction.

Reply:
I'm not really sure how to that! here we are just importing files from baseimage and checking the content! I'm wondering is there a way where we bundle something in the baseimage and extract and check!

Reply:
How about grepping `root` in the `/etc/passwd` file?

Reply:
i was hoping there might be something already in the busybox image you could check, but i guess there's not much in there.  can you at least confirm the extracted file is non-zero sized?

Reply:
> How about grepping root in the /etc/passwd file?

sounds good to me.

Reply:
Pushed the changes, hope it works fine 🤞